examples/thalamus_sample_dir/config/features/units.json

```diff
@@ -0,0 +1,44 @@
+{
+    "inv_time_to_first_spike": "Hz",
+    "inv_first_ISI": "Hz",
+    "inv_second_ISI": "Hz",
+    "inv_third_ISI": "Hz",
+    "inv_fourth_ISI": "Hz",
+    "inv_fifth_ISI": "Hz",
+    "inv_last_ISI": "Hz",
+    "time_to_last_spike": "ms",
+    "Spikecount": "constant",
+    "Spikecount_stimint": "constant",
+    "number_initial_spikes": "constant",
+    "ISI_semilog_slope": "ms",
+    "ISI_log_slope": "ms",
+    "check_AISInitiation": "ms",
+    "AP_amplitude": "mV",
+    "AP1_amp": "mV",
+    "AP2_amp": "mV",
+    "APlast_amp": "mV",
+    "AHP_depth_abs": "mV",
+    "AHP_depth_abs_slow": "mV",
+    "AHP_slow_time": "constant",
+    "AHP_depth": "mV",
+    "AHP_time_from_peak": "ms",
+    "AP_duration_half_width": "ms",
+    "AP_width": "ms",
+    "steady_state_voltage_stimend": "mV",
+    "steady_state_voltage": "mV",
+    "voltage_base": "mV",
+    "decay_time_constant_after_stim": "ms",
+    "sag_amplitude": "mV",
+    "sag_ratio1": "constant",
+    "sag_ratio2": "constant",
+    "ohmic_input_resistance": "mV/nA",
+    "ohmic_input_resistance_vb_ssse": "mV/nA",
+    "voltage_deflection_vb_ssse": "mV",
+    "voltage_after_stim": "mV",
+    "mean_frequency": "ms",
```

Comment:
This should be Hz.

Could you also change it in the units.json of sscx example, please?

Reply:
And only `mean_frequency` into "Hz"